crates/bevy_ecs/src/system/system_registry.rs

```diff
@@ -78,18 +94,6 @@ impl<I, O> std::fmt::Debug for SystemId<I, O> {
     }
 }
 
-impl<I, O> From<SystemId<I, O>> for Entity {
-    /// Transforms a [`SystemId`] into the [`Entity`] that holds the one-shot system's state.
```

Comment:
I think that these docs were generally quite helpful: perhaps we can incorporate them in `SystemId`'s docs?

Reply:
Rereading this I now understood why this was so verbose... the average bevy user is not as entity minded & needs to first have explained that entities can hold systems :)